.github/workflows/release.yml

```diff
@@ -130,22 +137,45 @@ jobs:
           trust_level: 5
 
       - name: Set default GPG key
-        run: |
-          gpg -K
-          gpg --list-secret-keys
-          echo "default-key ${{ steps.gpg.outputs.keyid }}" >> ~/.gnupg/gpg.conf
-          cat ~/.gnupg/gpg.conf
+        run: echo "default-key ${{ steps.gpg.outputs.keyid }}" >> ~/.gnupg/gpg.conf
```

Comment:
is there a reason to not update the prior commit with this simplified method rather than editing it here?